generics/services/entity-management.js

```diff
@@ -20,15 +20,14 @@ const entityManagementServiceUrl = process.env.ENTITY_MANAGEMENT_SERVICE_URL
  * @returns {JSON} - List of entity data.
  */
 
-const entityDocuments = function (filterData = 'all', projection = 'all', userToken) {
+const entityDocuments = function (filterData = 'all', projection = 'all') {
```

Comment:
@MallanagoudaB Hence you removed userToken from here, make changes in all places where the function is getting called